### ISSUE TITLE
It's not normal for the desktop settings to be affected by the application when it's closed

### ISSUE THREAD

**Issue by mosge** [iabem97/komorebi#93]
The desktop reset command must be included in the application so the desktop can be used normally when the app is closed, not only after uninstalling the application.

**Comment by RyanNerd** [comment]
Agreed. To get my desktop icons back after _stopping_ komorebi I had to run the script in the `README.md` file.
